django_project/ford3/templates/qualification.html

```diff
@@ -12,75 +13,181 @@
+                        </div>
+                    </div>
+                        <div class="row pb-3">
+                        {% for subject in qualification.entrance_req_subjects_list %}
+                            <div class="col-md-6">
+                                <b>Subject {{ forloop.counter }}: </b>{{ subject.name }} {{ subject.minimum_score }}%
+                            </div>
+                        {% endfor %}
+                        </div>
+                    {% endfor %}
+                </div>
+            </div>
+            <div class="row py-3 mx-0">
+                <div class='col-md-10 edu-underline px-0'>
+                    <h4 class="mt-3">4. Interests and Jobs</h4>
+                </div>
+                <div class='col-md-2'>
+                    <a class="edu-button edu-button-orange" href="#">Edit</a>
+                </div>
+            </div>
+            <div class='row py-3'>
                 <div class='col-md-12'>
+                    <div class="row pb-3">
+                        <div class="col-md-6">
+                            <b>Interests</b>
+                            <br />
+                            {% for  interest in qualification.interest_name_list %}
+                                {{  interest }}
+                                {% if forloop.counter < qualification.interest_name_list|length %}, {% endif  %}
+                            {% endfor %}
+                        </div>
+                        <div class="col-md-6">
+                            <b>Occupations</b>
+                            <br />
+                            {% for occupation in qualification.occupation_name_list %}
+                                {{  occupation }}
+                                {% if forloop.counter < qualification.occupation_name_list|length %}, {% endif  %}
+                            {% endfor %}
+                        </div>
+                    </div>
+                    <div class="row pb-3">
+                        <div class="col-md-6">
+                            <b>Does the qualification prepare for a critical skills occupation</b>
+                            {% if qualification.critical_skill %}
+                                &#9989
+                            {% else %}
+                                &#10060
+                            {% endif %}
+                        </div>
+                        <div class="col-md-6">
+                            <b>Does the qualification prepare for a green occupation</b>
+                            {% if qualification.green_occupation  %}
+                                &#9989
+                            {% else %}
+                                &#10060
+                            {% endif %}
+                        </div>
+                    </div>
+                    <div class="row pb-3">
```

Comment:
it's weird to re-create a new `.row`.
And at the end, it doesn't look good
do something like
```
<p>
<b>Does the qualification prepare for a critical skills occupation</b>
<br />
...
</p>
```

Reply:
using `<p>` will result:

![Screen Shot 2019-05-13 at 3 56 33 PM](https://user-images.githubusercontent.com/4602383/57612213-9a913180-759e-11e9-92f6-8b554c926399.png)

the use of `.row` is to make sure that they are in a row.
the `interests` and `occupations` can be of different height.
do you still want to use `<p>` instead?

Reply:
I dont know Boney, does it look good to you?
Maybe I will get rid of the loooong question `Does the qualification....`
Just 'critical skills', ...